README.md

```diff
@@ -32,6 +32,8 @@ cerberus:
         -    openshift-sdn
         -    openshift-ovn-kubernetes
     cerberus_publish_status: True                        # When enabled, cerberus starts a light weight http server and publishes the status
+    data_inspection: True                                # When enabled, cerberus collects logs, events and metrics of failed components
```

Comment:
Inspection is a heavy operation, let's turn it off by default. Thoughts?

Reply:
Makes sense. Let's turn it off by default.

Reply:
I have updated it.